src/features/parameters/dynamic-simulation/dynamic-simulation-parameters-form.tsx

```
@@ -36,93 +36,106 @@ type DynamicSimulationFormProps = {
-                                evaluateFilterFetcher={evaluateFilterFetcher}
-                            />
-                        </TabPanel>
-                    </Grid>
-                    {renderActions?.(onError)}
-                </Grid>
-            ) : (
-                <LinearProgress />
-            )}
-        </>
+        <ParameterLayout
+            title={'DynamicSimulation'}
```

ESLint

[error] Curly braces are unnecessary here.

(react/jsx-curly-brace-presence)